doc/manual/src/installation/installing-binary.md

```diff
@@ -191,8 +202,7 @@ and `/etc/zshrc` which you may remove.
 <!-- Note: anchors above to catch permalinks to old explanations -->
 
 We believe we have ironed out how to cleanly support the read-only root
-on modern macOS. New installs will do this automatically, and you can
-also re-run a new installer to convert your existing setup.
+on modern macOS. New installs will do this automatically.
```

Comment:
To clarify, the old statement was true for a while (and is technically true of the volume-setup parts of the macOS install), but installs can now break with a populated store since `rsync` was swapped out for `cp` in #5150. (I opened https://github.com/NixOS/nix/issues/6679 so that we have a chance to avoid losing the knowledge that it once worked this way.)

We've been getting a steady trickle of these reports since ~December (https://github.com/NixOS/nix/search?o=desc&q=%22cp%3A+cannot+overwrite+directory%22&s=created&type=issues). Not that I think fixing the language will actually help (I imagine most of these people are just _reasoning_ that a reinstall can/would/should fix whatever ails them).